packages/next/build/webpack/loaders/next-middleware-ssr-loader/index.ts

```diff
@@ -1,6 +1,28 @@
 import loaderUtils from 'next/dist/compiled/loader-utils'
 import { getStringifiedAbsolutePath } from './utils'
 
+const fallbackDocumentPage = `
+import { Html, Head, Main, NextScript } from 'next/document'
+
+function Document() {
+  return (
+    jsx(Html, null, 
+      jsx(Head),
+      jsx('body', null,
+        jsx(Main),
+        jsx(NextScript),
+      ),
+    )
+  )
+}
+`
+
+const fallbackAppPage = `
+function App({ Component, pageProps }) {
+  return jsx(Component, pageProps)
+}
+`
```

Comment:
For now we don't have to use the inlined `_document` and `_app`, we can use the default ones (e.g. https://github.com/vercel/next.js/blob/a25c1dd15ac0d711a0ff90651e62627408ecf752/packages/next/pages/_document.tsx) even if they're class components.